src/Components/blazor/version.props

```
@@ -0,0 +1,17 @@
+<Project>
+  <PropertyGroup>
+    <!-- When editing any version prefix here, remember also to update
+         tooling\Microsoft.VisualStudio.BlazorExtension\Properties\AssemblyInfo.cs
+         to include the new version in its upper bounds -->
+    <IsFinalBuild>false</IsFinalBuild>
+    <VersionPrefix>0.8.0</VersionPrefix>
+    <VersionSuffix>preview1</VersionSuffix>
```

```suggestion
    <VersionSuffix>preview</VersionSuffix>
```

We have stopped adding X as in "previewX". The guidance is to build as `-preview-$(BuildNumberSuffix)`. See this repo's root version.props file for more details.

---

Done